src/micro_ros_arduino.h

```diff
@@ -102,10 +102,13 @@ struct micro_ros_agent_locator {
 };
 
 static inline void set_microros_wifi_transports(char * ssid, char * pass, char * agent_ip, uint agent_port){
-	while (WiFi.begin(ssid, pass) != WL_CONNECTED)
-	{
-    	delay(500);
-	}
+
```

Comment:
Is this checked in other boards?

Reply:
Yes, fixed a bug here: https://github.com/micro-ROS/micro_ros_arduino/pull/680/commits/de6bc5bea0d6cff7fd7f9d7fe3446cd687aaaa55